Src/Newtonsoft.Json/JsonReader.cs

```diff
@@ -51,7 +51,7 @@ public abstract class JsonReader : IDisposable
         protected internal enum State
         {
             /// <summary>
-            /// The <see cref="Read"/> method has not been called.
+            /// A <see cref="JsonReader"/> read method has not been called.
```

Comment:
This is one of three similar comments that appear on these code elements:

* `JsonReader.State.Start`
* `WriteState.Start`
* `JsonToken.None`

The thing here is that a whole group of methods is referred to. That's why I changed "the" to "a" and removed the reference to one particular method (even though that appears to be the one that always gets called internally).